ports/metal-cpp/portfile.cmake

```
@@ -0,0 +1,21 @@
+vcpkg_download_distfile(
+    archive
+    URLS https://developer.apple.com/metal/cpp/files/metal-cpp_macOS12_iOS15.zip
+    FILENAME metal-cpp.zip
+    SHA512 dabb4109c7bf283288b5b3bd392892a7a52ad13b4d53a72a117a852f54b0a82871bec3e55c8493d8048365839bd1be37d72f872041f43db314622bb4a983921f
+)
+
+vcpkg_extract_source_archive(

```

```suggestion
vcpkg_extract_source_archive_ex()

```
`vcpkg_extract_source_archive ` are deprecated. Please use `vcpkg_extract_source_archive_ex()` instead